spec/intl.html

```diff
@@ -615,7 +617,7 @@ <h1>PartitionDateTimeRangePattern ( _dateTimeFormat_, _x_, _y_ )</h1>
         1. If _dateFieldsPracticallyEqual_ is *true*, then
           1. Let _pattern_ be <del>_dateTimeFormat_.[[Pattern]]</del><ins>_x_.[[pattern]]</ins>.
           1. Let _patternParts_ be PartitionPattern(_pattern_).
-          1. Let _result_ be ? FormatDateTimePattern(_dateTimeFormat_, <ins>_pattern_,</ins> _patternParts_, _x_<ins>.[[epochNanoseconds]]</ins>, *undefined*).
+          1. Let _result_ be ? FormatDateTimePattern(_dateTimeFormat_, <ins>_pattern_,</ins> _patternParts_, _x_<ins>.[[epochNanoseconds]]</ins>, *undefined*, <ins>_x_.[[timeZone]]</ins> ).
```

Comment:
_x_.[[TimeZone]] may be undefined here if _x_ and _y_ were not ZonedDateTime instances.